src/js/api/auth/logout.test.js

```diff
@@ -0,0 +1,12 @@
+import { logout } from './logout';
+import { LsMock, TEST_TOKEN } from './login.test';
+
+global.localStorage = new LsMock();
+
+describe('logout', () => {
+  it('Removes the valid user token on logout', () => {
+    localStorage.setItem('token', JSON.stringify(TEST_TOKEN));
+    logout();
+    expect(localStorage.getItem('token').toEqual(null));
```

Comment:
Also, this test is failing. Pls, have a look
![failing test](https://user-images.githubusercontent.com/25647480/204097506-57115616-7135-4b9a-a181-f3717d0101c1.PNG)